### ISSUE TITLE
Invalid Image URLs on Studio Page

### ISSUE THREAD

**Issue by shengdade** [shinework/photoshot#54]
Hello,

I've encountered an issue while navigating through the studio page. The photo shots displayed on the page are linked to invalid URLs, thus the images are not loading as expected. 

The images are hosted on Replicate CDN and it seems like these images were cleaned up after some time, leading to the invalid URLs. This might be the reason why the images are no longer available on the studio page.

Is there any mechanism we can implement to persist these images or prevent them from being cleaned up? 

Thanks!

**Comment by lopezi** [comment]
Yes I also came across this one which is not very user friendly... is there a way to configure this to archive the images on the studios? would be really great!

<img width="1101" alt="Screen Shot 2023-10-03 at 21 47 05" src="https://github.com/shinework/photoshot/assets/388476/2dba176a-3f04-418a-85b6-4564eb2f3a4c">

**Comment by lopezi** [comment]
could re-install docker-compose cause this? sorry if it is an obvious question but still wrapping around the architecture

**Comment by lopezi** [comment]
replicate is deleting the predictions after one hour. this app needs to store the predictions, for example, on s3, since it is already using it for the training or it renders useless as a production app. anyone interested in collaborating with this? I can help with any UI tasks but handling data is not my expertise.
